proxy/README.md

```
@@ -0,0 +1,57 @@
+   ```bash
+   openssl genrsa -out server.pem 2048
+   ```
+
+4. **Create Server Certificate Signing Request (CSR)**:
+
+   ```bash
+   openssl req -new -key server.pem -out server.csr -subj "/CN=alpha-sepolia.starknet.io"
+   ```
+
+5. **Sign Server CSR with CA Certificate**:
+   ```bash
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)